commands/login_test.go

```diff
@@ -5,6 +5,8 @@ import (
 	"strings"
 	"testing"
 
+	"github.com/10gen/realm-cli/utils/telemetry"
+
```

Comment:
starting to wish i didn't choose to die on this hill, but a re-org here to fit our standards would be much appreciated